src/shared/components/simpleTable/styles.scss

```diff
@@ -5,6 +5,14 @@ th {
   background-color: rgb(255,255,255); // solid background
 }
 
+tbody>tr:nth-of-type(odd).highlight {
+  background-color: #E0E000 !important;
+}
+
 tbody>tr:nth-of-type(even) { // solid background - hard coded hack, react-bootstrap .table-striped makes ODD # rows grey
   background-color: rgb(255,255,255);
+}
+
+tbody>tr:nth-of-type(even).highlight {
+  background-color: #E9E900;
```

Comment:
it seems that it's not necessary because:
(1) bootstrap .table-striped doesn't define any style for the even rows, which is why we were having that problem with no background for those rows
(2) because it has `.highlight` it has higher specificity than the rule above

but I guess there's no harm in adding it? I'm not experienced with CSS so whatever you think.